Okta/OktaAuth/OktaAuthStateManager.swift

```diff
@@ -154,7 +176,7 @@ extension OktaTokenManager {
         let authStateData = NSKeyedArchiver.archivedData(withRootObject: self)
         do {
             try OktaKeychain.set(
-                key: OktaTokenManager.secureStorageKey,
+                key: OktaAuthStateManager.secureStorageKey,
```

Comment:
Currently we have hardcoded value for the key - "OktaAuthStateTokenManager"
I would suggest additional enhancement where we make composite key and don't use hardcoded keys, let's say concatenate `issuer` and `clientId`. This should give us unique key for the storage. With that approach we can guarantee that `OktaAppAuth` instance loads correct `OktaAuthStateManager` data from keychain in multi account applications(this should also fix issue with `signOut`)
@anastasiiaiurok-okta , please decide whether you want to handle this in current PR or I can create separate Jira ticket

@robertdamphousse-okta , @nbarbettini , to review

Reply:
@IldarAbdullin-okta Let's do this in a separate PR.